interface/usergroup/user_info.php

```diff
@@ -1,15 +1,29 @@
 <?php
+/**
+ * User password change tool
+ *
+ * @package     OpenEMR
+ * @link        http://www.open-emr.org
+ * @author      Roberto Vasquez <robertogagliotta@gmail.com>
+ * @copyright   Copyright  (C) 2017 Roberto Vasquez <robertogagliotta@gmail.com>
+ * @license     https://github.com/openemr/openemr/blob/master/LICENSE CNU General Public License 3
+ *
+*/
 include_once("../globals.php");
 include_once("$srcdir/auth.inc");
+
+use OpenEMR\Core\Header;
+
 if($GLOBALS['use_active_directory']) {
     exit();
 }
 ?>
 <html>
 <head>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<?php Header::setupHeader(); ?>
+<title><?php echo xlt('Password Change'); ?></title>
```

Comment:
Suggest using "change password" here instead. It  is a better flow, grammatically